fix TestISequencer_FlushPermanentlyFails

https://github.com/voedger/voedger/actions/runs/17339742953/job/49231981880
```
  --- FAIL: TestISequencer_FlushPermanentlyFails (2.34s)
      test_utils.go:30: 
          	Error Trace:	/home/runner/work/voedger/voedger/pkg/isequencer/test_utils.go:30
          	            				/home/runner/work/voedger/voedger/pkg/isequencer/isequencer_test.go:968
          	Error:      	Should be true
          	Test:       	TestISequencer_FlushPermanentlyFails
```

--- a/pkg/isequencer/consts.go
+++ b/pkg/isequencer/consts.go
@@ -14,7 +14,7 @@ const (
 	DefaultMaxNumUnflushedValues             = 500
 	defaultBatcherDelayOnToBeFlushedOverflow = 5 * time.Millisecond
 	baseRetryDelay                           = 500 * time.Millisecond
-	maxRetryDelay                            = 10 * time.Second
+	maxRetryDelay                            = baseRetryDelay
 )
 
 const (

--- a/pkg/isequencer/test_utils.go
+++ b/pkg/isequencer/test_utils.go
@@ -9,12 +9,11 @@ import (
 	"context"
 	"math"
 	"testing"
-	"time"
 
 	"github.com/stretchr/testify/require"
 )
 
-var actualizationTimeoutLimit = 1 * time.Second
+const actualizationTimeoutLimit = maxRetryDelay
 
 // waitForActualization waits for actualization to complete by repeatedly calling Start
 func WaitForStart(t *testing.T, seq ISequencer, wsKind WSKind, wsID WSID, shouldBeOk bool) PLogOffset {

